packages/documentation/src/stories/internet-header/components/breadcrumbs/overrides/breadcrumbs-config.docs.mdx

```diff
@@ -0,0 +1,14 @@
+import { Meta, Canvas, Controls } from '@storybook/blocks';
+import * as BreadcrumbStories from './breadcrumbs-config.stories';
+
+<Meta of={BreadcrumbStories} />
+
+# Breadcrumbs with custom items
+
+<div className="lead">When you need to add your own breadcrumb items.</div>
+
+<Canvas of={BreadcrumbStories.Default} />
+
+<div className="container">
+  <Controls of={BreadcrumbStories.Default} />
+</div>
```

Comment:
```suggestion
<Controls of={BreadcrumbStories.Default} />
```

Reply:
This is still a problem.
With the `.container` wrapper:
![image](https://github.com/swisspost/design-system/assets/9716662/ae5b34d5-0651-4522-be2b-051f02c07c46)
Without the `.container` wrapper:
![image](https://github.com/swisspost/design-system/assets/9716662/b82db9f9-fd82-46e8-9675-2f08bcd79cde)